docs/designs/scrapping-workflow-phase-1-design.md

```diff
@@ -0,0 +1,67 @@
+## Scrapping workflow phase 1 design 
+
+**General requirements**
+* Workflow can support multiple sources/multiple scraping strategies
+* Workflow can be scaled easily
+* Workflow is observable
+
+**What currently exists**
+* From [this Design Doc](https://www.notion.so/de8ece87473b4f37a0ea037d607d910a?v=1bec41f87a6b42acb639d348d2282195&p=6d6b1797931e465d9cf6f5d4641c911e), Angostura can be used to publish and persist data to google BigQuery.
+* Sambil has been packaging their code as a library: c4v-py (this repo), which can be imported and used elsewhere.
+
+**Proposed design** 
+
+_Goal_: Leverage existing angostura infrastructure as much as possible
+![image](../images/scrapping-workflow-phase-1-design.png)
+
+1. A custom written data uploader processes NGO submitted data and emits the events to the Angostura proxy gate.
```

Comment:
Diego Garcia used this [python script](https://colab.research.google.com/drive/1Hj0Z1JHBIBaiG4v87WC3yJaAWCneD7Ik?usp=sharing) to upload last NGO dataset.  Miht be a good idea to use it as a starting point?

Reply:
I've had a chat with Erika with angostura and she suggested that we might not even need this. 
I think they have a way of moving data from google sheets to big query, so we could start there. 
I'll be investigating ths

Reply:
Keep us posted (and whatever the solution, we can add it to the design doc itself)!

Reply:
Edited the design doc with what I learned from her, it's not too big a change.